lib/generators/hyrax/work_resource/work_resource_generator.rb

```diff
@@ -33,6 +33,18 @@ def create_model_spec
       rspec_installed?
   end
 
+  def create_indexer
+    template('indexer.rb.erb', File.join('app/indexers/', class_path, "#{file_name}_indexer.rb"))
+  end
+
+  def register_indexer
+    config = 'config/initializers/indexers.rb'
+    register_line = "  Hyrax::ValkyrieIndexer.register #{class_name}Indexer, as_indexer_for: #{class_name}\n"
+    inject_into_file config, after: "# Register Indexers\n" do # rubocop:disable Style/MultilineIfModifier
```

Comment:
could avoid the rubocop exception with a guard clause?

```ruby
return if File.read(config).include?(register_line)
inject_into_file config, after: "# Register Indexers\n" { register_line }
```

???
```

Reply:
Oh that should work, I tried various permutations that all had different rubocop errors!